### ISSUE TITLE
Quarto and flextable_to_rmd: Error in `.make_numeric_version()`

### ISSUE THREAD

**Issue by Steinthal** [quarto-dev/quarto-cli#9583]
### Bug description

"`flextable_to_rmd()`" throws 

"Error in `.make_numeric_version()`:
! invalid non-character version specification 'x' (type: double)"

### Steps to reproduce

````
---
title: "Test Flextable"
format: pdf
---

```{r}
#| warning: false
#| message: false
#| echo: false

library(flextable)
library(knitr)
```

```{r}
#| warning: false
#| message: false
#| echo: false
#| output: asis

ft <- flextable(head(mtcars)) |>
  autofit() |>
  flextable_to_rmd()
```
````

### Expected behavior

A pdf file is created and displayed.

### Actual behavior

The system issues this information:

"processing file: test_flextable.qmd
  |..........................................          |  80% [unnamed-chunk-2]
Quitting from lines  at lines 17-30 [unnamed-chunk-2] (test_flextable.qmd)

Error in `.make_numeric_version()`:
! invalid non-character version specification 'x' (type: double)

Backtrace:
 1. flextable::flextable_to_rmd(autofit(flextable(head(mtcars))))
 2. flextable:::is_in_quarto()
 4. base::Ops.numeric_version(...)
 5. base::as.numeric_version(e1)
 6. base::numeric_version(x)
 7. base::.make_numeric_version(x, strict, .standard_regexps()$valid_numeric_version)
                                                                                                            
Execution halted"

Not sure if this is relevant: This happens since upgrading RStudio and R to below stated versions.

### Your environment

- IDE Version 2024.04.0+735 (2024.04.0+735)
- R version: 4.4.0
- macOS: 14.5 Beta (23F5074a)

### Quarto check output

```
Quarto 1.5.32
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.13: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.32
      Path: /Applications/quarto/bin

[✓] Checking tools....................OK
      Chromium: 869685
      TinyTeX: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /opt/homebrew/bin
      Version: undefined

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.3
      Path: /opt/homebrew/opt/python@3.12/bin/python3.12
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

[✓] Checking R installation...........OK
      Version: 4.4.0
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/library
      knitr: 1.46
      rmarkdown: 2.26

[✓] Checking Knitr engine render......OK
```

**Comment by Steinthal** [comment]
After downgrading R to 4.3.3, the PDF file is generated as expected.

**Comment by mcanouil** [comment]
That's not a Quarto issue.
Please report to flextable repository.

**Comment by Steinthal** [comment]
Will be solved with flextable 0.9.6
